api/kernel/pci_manager.hpp

```diff
@@ -37,6 +38,9 @@ class PCI_manager {
   using BLK_driver = delegate< std::unique_ptr<hw::Block_device> (hw::PCI_Device&) >;
   static void register_blk(uint16_t, uint16_t, BLK_driver);
 
+  using SND_driver = delegate< std::unique_ptr<hw::Audio_device> (hw::PCI_Device&) >;
```

Comment:
Hmm i wonder if we could do this in a more "generic" way to avoid having to add these in the header.

Reply:
Alfred is doing the work now, this is the current dev, so Im not trying to reinvent here